src/test/java/org/kie/processmigration/service/PlanServiceImplTest.java

```diff
@@ -42,13 +42,15 @@ class PlanServiceImplTest {
     @Inject
     PlanService planService;
 
-    @Inject
-    Flyway flyway;
-
-    @AfterEach
+    @BeforeEach
     void cleanUp() {
-        flyway.clean();
-        flyway.migrate();
+        planService.findAll().forEach(plan -> {
+            try {
+                planService.delete(plan.getId());
+            } catch (PlanNotFoundException e) {
+                // ignore
+            }
+        });
```

Comment:
So the other 2 tests are fine without any cleaning?

Reply:
they're fine, but I agree with you that it would be good to make a DB cleanup as well, just in case. Thank you.
To double check I configured surefire plugin to execute test randomly in my local to make sure there is no leftover in the DB.